experimental/tfhub/tests/test_tfhub.py

```diff
@@ -56,3 +82,27 @@ def test_ssd(spark: SparkSession):
     """
     )
     assert result.count() == 1
+
+
+def test_multi_pics_ssd(spark: SparkSession):
+    spark.udf.register("to_image2", to_image)
```

Comment:
why we need to_image2?

Reply:
nit: how about register all udfs or the used udfs here

https://github.com/eto-ai/rikai/blob/fceff116560dc16c1f588cf25ea208fe0b1220e6/experimental/tfhub/tests/conftest.py#L22

Reply:
updated